android/app/src/main/res/layout/item_schedule.xml

```diff
@@ -13,16 +13,34 @@
     android:paddingStart="@dimen/padding_extra_large"
     android:paddingTop="@dimen/padding_medium">
 
-    <TextView
-        android:id="@+id/slot_title"
-        android:layout_width="wrap_content"
+    <LinearLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:ellipsize="end"
-        android:paddingBottom="@dimen/padding_small"
-        android:paddingTop="@dimen/padding_small"
-        android:textColor="@color/black"
-        android:textSize="@dimen/text_size_extra_large"
-        tools:text="Slot Title" />
+        android:orientation="horizontal">
+
+        <TextView
+            android:id="@+id/slot_title"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="13"
+            android:ellipsize="end"
+            android:paddingBottom="@dimen/padding_small"
+            android:paddingTop="@dimen/padding_small"
+            android:textColor="@color/black"
+            android:textSize="@dimen/text_size_extra_large"
+            tools:text="Slot Title" />
+
+        <ImageButton
+            android:id="@+id/slot_bookmark"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:background="?attr/selectableItemBackgroundBorderless"
+            android:contentDescription="@string/session_bookmark_status"
+            android:padding="@dimen/padding_small"
+            android:tint="@color/black"
+            app:srcCompat="@drawable/ic_bookmark_border_white_24dp" />
+    </LinearLayout>
```

Comment:
Instead of 13 and 1 weights (which are hardcoded), you can use weight 1 for textview and wrap_content width for ImageButton

Reply:
Ok :+1: